test/selenium/selenium-machines-disks.py

```diff
@@ -139,7 +139,7 @@ def testDetachDiskVmOff(self):
         self.click(self.wait_css('#vm-{}-disks-adddisk-dialog-add'.format(name), cond=clickable))
         self.wait_css('#vm-{}-disks-vda-device'.format(name))
         self.click(self.wait_css('#vm-{}-disks-vda-detach'.format(name), cond=clickable))
-        self.wait_css('vm-{}-disks-vda-device'.format(name), cond=invisible)
+        self.wait_css('#vm-{}-disks-vda-device'.format(name), cond=invisible)
```

Comment:
This is a fix unrelated to this commit and should be separated.

Reply:
Separate it to another commit